source/code/scxsystemlib/common/scxostypeinfo.cpp

```diff
@@ -52,7 +52,7 @@ using namespace SCXCoreLib;
 
 namespace {
 
-#if defined(linux)
+#if defined(linux) && defined(PF_DISTRO_SUSE)
```

Comment:
defined(linux) doesn't include PF_DISTRO_SUSE ?

I think the intent was that this ran on all Linux platforms. But now it only runs on SUSE, and I don't think this is correct.